app/controllers/helpers/TaxReturnViewModel.scala

```diff
@@ -74,7 +74,12 @@ case class TaxReturnViewModel (
   }
 
   // Show or hide edit links
```

Comment:
should this comment be here or for the defs below?

Reply:
for the defs below. Thanks